server/exercise/src/main/scala/com/eigengo/lift/exercise/MultiPacket.scala

```diff
@@ -4,10 +4,10 @@ import scodec.bits.BitVector
 
 case class PacketWithLocation(sourceLocation: SensorDataSourceLocation, payload: BitVector)
 
-case class MultiPacket(packets: List[PacketWithLocation]) {
+case class MultiPacket(timestamp: Long, packets: List[PacketWithLocation]) {
```

Comment:
I understand the reason for transmitting a long value on the wire, however, is there any utility in decoding this value into a semantically meaningful timestamp value (e.g. a Jodatime DateTime)?